lint/lint.py

```diff
@@ -226,6 +226,23 @@ def check_parsed(repo_root, path, f, css_mode):
     if source_file.type == "visual" and not source_file.name_is_visual:
         return [("CONTENT-VISUAL", "Visual test whose filename doesn't end in '-visual'", path, None)]
 
+    for reftest_node in source_file.reftest_nodes:
+        href = reftest_node.attrib.get("href", "")
+        path_parts = [repo_root]
+        if href[0] != "/":
```

Comment:
I think we should at least be using a URL parser here, and detecting whether there is a scheme or authority present (and probably give a different error in that case).